tests/storage/utils.py

```diff
@@ -487,3 +483,10 @@ def clean_up_multiprocess(processes, object_list):
             print(f"Error killing process {process}, associated with {object_name}: {e}")
         finally:
             process.close()
+
+
+def get_data_import_cron_conditions(namespace, name):
+    data_import_cron = DataImportCron(namespace=namespace, name=name)
+    for condition in data_import_cron.instance.status.get("conditions"):
+        if condition["type"] == "UpToDate":
+            return condition["status"]
```

Comment:
If we will need this function, please move it to `tests/storage/data_import_cron/utils.py`